docs/index.rst

```diff
@@ -2,74 +2,162 @@
 Dask-Jobqueue
 =============
 
-*Easy deployment of Dask Distributed on job queuing systems like
-PBS, Slurm, and SGE.*
+*Easily deploy Dask on job queuing systems like PBS, Slurm, MOAB, and SGE.*
```

Comment:
Just curious, why did you remove _distributed_, and only kept Dask?

Reply:
The name of the project is Dask.  It happens to have a local and a distributed scheduler but these are, from my perspective, not part of the project name.  The fact that there is a repository named `distributed` is a historical artifact and something that I prefer to hide.

Reply:
Right, I totally support this. distributed not really being part of dask is something I've been told when evangelizing dask around me. Much better if they are really shown as one.